taf/developer_tool.py

```diff
@@ -503,7 +514,7 @@ def generate_repositories_json(
         url = target_repo.get_remote_url()
         if url is None:
             if targets_relative_dir is not None:
-                url = Path(target_repo.repo_path).relative_to(targets_relative_dir)
+                url = Path(os.path.relpath(target_repo.repo_path, targets_relative_dir))
```

Comment:
Was this a windows issue with paths?

Reply:
Not sure if it is Windows specific problem, but `relative_to` does not work as well as `relpath`. I remember having that issue before